docs/pages/base/api/table-pagination-unstyled.json

```diff
@@ -40,7 +40,35 @@
     }
   },
   "name": "TablePaginationUnstyled",
```

Comment:
Not sure why some components had no props extracted before, since I kept a similar parsing approach.
I guess that's linked to my `resolveExportSpecifier` which handles the following pattern:

```tsx
const XXX = () => {}

export default XXX
```

Reply:
Base components don't usually have the `classes` prop. TablePaginationUnstyled seems to differ in this matter, but thanks to your changes, we identified it.

Reply:
Maybe we could crash or at least warn when there is a `classes` prop but no `classes` extracted on `yarn docs:api:build`